source/03-specifications/01-counter-reports-for-libraries.rst

```diff
@@ -64,7 +64,13 @@ Table 3.a (below): COUNTER Reports
      - Data_Repository*\ |br|\ |lb|
        Multimedia\ |br|\ |lb|
        Repository\ |br|\ |lb|
-       Scholarly_Collaboration_Network
+       Scholarly_Collaboration_Network\ |br|\ |lb|
+       Four other Host_Types are encouraged but not required to provide Item Reports, specifically\ |br|\ |lb|
+       Aggregated_Full_Content\ |br|\ |lb|
+       eBook\ |br|\ |lb|
+       eBook_Collection\ |br|\ |lb|
+       eJournal
+
```

Comment:
So far everything that is optional is described below the tables, since the tables only list things that are required (which is what the paragraph before the table states). I would suggest to at least add a blank like before this part if it should stay within the table.

What about other Host_Type like Full_Content_Database and Multimedia_Collection?

Reply:
You've removed this part from Table 3.a (and replaced it with a separate section), but not from Table 4.n. Was that intentional?

Reply:
edited